platforms/python/buildPython.sh

```diff
@@ -46,17 +45,12 @@ buildPython() {
 		else
 			dockerFile="$pythonPlatformDir/$dockerFile"
 		fi
-
-		if [ -z "$pipVersion" ]; then
```

Comment:
Made this change since we have come across a requirement where we lock into specific version of PIP for a particular Python version.